user/languages/objective-c.md

```diff
@@ -23,15 +23,92 @@ NDA imposed on them. We do test them internally, and our goal is to make new
 versions available the same day they come out. If you have any further questions
 about Xcode pre-release availability, send us an email at support@travis-ci.com.
 
-### Xcode 7
+### Xcode 7.1
 
-Xcode 7 GM is available by adding `osx_image: xcode7` to your .travis.yml.
+Xcode 7.1 GM is available by adding `osx_image: xcode7.1` to your .travis.yml.
 
 Our Xcode 7 image has the following SDKs preinstalled:
 
 - macosx10.11
 - iphoneos9.0
```

Comment:
`iphoneos9.0` and `iphonesimulator9.0` are not installed, so I would remove them from this list.

`iphoneos9.1` and `appletvsimulator9.0` should be added to the list.

See https://magnum.travis-ci.com/travis-pro/xcode7-sample-project/jobs/25873523#L118